### ISSUE TITLE
Accessibility of composite questions

### ISSUE THREAD

**Issue by jefftudorbenbria** [surveyjs/survey-library#5985]
The issue is reproducible in the demo from the survey js website when viewing this from the IOS VoiceOver screen reader. https://surveyjs.io/survey-creator/examples/add-full-name-to-form-template/reactjs

### Are you requesting a feature, reporting a bug or asking a question?
Bug

### What is the current behavior?
The IOS VoiceOver screen reader focuses on the background of the composite question and skips the text entry fields.

### What is the expected behavior?
The  expected behaviour is that the text entry fields would be navigate-able from the screen reader. When navigating the survey the screen reader should focus each of the input fields.

### How would you reproduce the current behavior (if this is a bug)?
Go to this survey-js demo (https://surveyjs.io/survey-creator/examples/add-full-name-to-form-template/reactjs) and enter the preview mode. Then enable the screenreader on your IOS device (VoiceOver) and try to navigate through the survey.

### Specify your

* browser: safari
* browser version:  16.00
* surveyjs platform (angular or react or jquery or knockout or vue): latest
* surveyjs version: latest

**Comment by jefftudorbenbria** [comment]
FYI I experienced this same issue with the single input question type and the comment question type

**Comment by dmitrykurmanov** [comment]
reproduced: we can't navigate to elements via "tab"

**Comment by JaneSjs** [comment]
Hello @jefftudorbenbria,
I tested the Full Name demo using the VoiceOver iOS screenreader. Please refer to my screencast: 
https://user-images.githubusercontent.com/22372972/232407609-6d4017d1-4ee4-4a15-aea5-e3a069dd3e26.mov

It seems that the screeenreader can correctly announce the composide question's fields. Would you please clarify how to reproduce the issue?

I look forward to your reply.

**Comment by lucaspawley-benbria** [comment]
Hello @JaneSjs 

I was able to reproduce it by following the steps outlined below:

1. navigate to the survey preview with voice over active
2. click on any part of the survey to start the voiceover
3. swipe to the right (or use your configured gesture) to go to the next element until you reach the composite question background
4. when i went to the next element, it skipped the inputs and went to the complete button.

the expected behavior being that swiping while on the question moves the voiceover to the text input

thank you

**Comment by naerymdan** [comment]
Seems related to https://github.com/surveyjs/survey-library/issues/6058

**Comment by dmitrykurmanov** [comment]
We will try to reproduce the issue by provided steps.

**Comment by JaneSjs** [comment]
Hello @lucaspawley-benbria,
Thank you for the update. For us to move forward, we would ask you to share a problematic survey JSON for research. Also, please specify the device you're using.

I look forward to your reply.

**Comment by lucaspawley-benbria** [comment]
hello,

we were able to reproduce the issue on an Iphone 12 on version 16.4.1, and an iPad (9th generation) on version 15.6.1

 the issue is reproducible on the example above, so the following json should work
`{
 "pages": [
  {
   "name": "page1",
   "elements": [
    {
     "type": "fullname",
     "name": "question1"
    }
   ]
  }
 ]
}`

thank you
Lucas

**Comment by dmitrykurmanov** [comment]
We tried on Mac. We will try to reproduce the problem on Ipad.

**Comment by JaneSjs** [comment]
Hello Everyone,
We reproduced this issue on an iPad. We will research this issue and update this thread once we have any news to share.

Please stay tuned.

**Comment by JaneSjs** [comment]
Hello,
We were able to reproduce the issue, and it appears to be related to the role='textbox' attribute of survey questions, as you correctly noticed. Our team is currently researching this issue and making the required changes in our code to ensure that survey questions are accessible for the VoiceOver tool on iOS devices. We apologize for any inconvenience this may cause, and we cannot provide a specific timeframe for when the issue will be resolved, as the changes required may be significant.

In the meantime, we recommend applying the following temporary solution to remove the `role='textbox'` attribute for survey questions, allowing the VoiceOver tool to correctly process them:
```
survey.onAfterRenderSurvey.add(() => {
  const surveyElement = document.getElementById("surveyElement");
  const roleTextboxNodes = surveyElement.querySelectorAll("[role='textbox']");
  roleTextboxNodes.forEach((node) => {
    node.removeAttribute("role");
  });
});
Thank you for bringing this issue to our attention, and please look forward to updates from us regarding a permanent solution.

Thank you

**Comment by JaneSjs** [comment]
Hello Everyone,
We are currently research this issue and will update the following thread once we have any news to share: https://github.com/surveyjs/survey-library/issues/6058.

Thank you
